src/test/java/tests/LoginPageTest.java

```diff
@@ -0,0 +1,57 @@
+package tests;
+
+import helpers.UserCreator;
+import model.User;
+import org.testng.annotations.Test;
+import pages.LoginPage;
+import pages.ProductsPage;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class LoginPageTest extends BaseTest {
+
+    @Test(priority = 0, description = "Standard user should login")
+    public void standardUserShouldLogin() {
+        LoginPage loginPage = new LoginPage(driver);
```

Comment:
I guess it's better to store the driver instance in a separate class and retrieve it for Page, Step. No reason to manage it from the Test level